README.md

```diff
@@ -27,29 +27,39 @@ one you should use, refer to the [`argon2` repo][argon2].
 ### Hashing a password
 
 ```javascript
+var crypto = require('crypto');
 var argon2i = require('argon2-ffi').argon2i;
 // var argon2d = require('argon2-ffi').argon2d; if you'd like to use argon2d
 
 var password = new Buffer('password1');
-var salt = new Buffer('saltysalt');
-argon2i.hash(password, salt, function(err, res) {
-  console.log(res); // $argon2i$v=19$m=4096,t=3,p=1$c2FsdHlzYWx0$oG0js25z7kM30xSg9+nAKtU0hrPa0UnvRnqQRZXHCV8
+crypto.randomBytes(128, function (err, salt) {
+  argon2i.hash(password, salt, function(err, res) {
+    console.log(res); // $argon2i$v=19$m=4096,t=3,p=1$c2FsdHlzYWx0$oG0js25z7kM30xSg9+nAKtU0hrPa0UnvRnqQRZXHCV8
```

Comment:
I kept the style of ignoring errors since this is an example. If you want me to change that let me know.